doc/RELEASE_NOTES.md

```diff
@@ -0,0 +1,828 @@
+
+### User Orchestration of "Tool Meister" Sub-System
+
+Container images are provided for the constituent components of the Tool Meister sub-system, the Tool Meister image and the Tool Data Sink image.  The images allow for the orchestration of the Tool Meister sub-system to be handled by the user instead of automatically by the pbench-agent.
+
+### The "Tool Meister" Sub-System with No Tools
+
+While this is not a new feature of the Pbench Agent, it is worth noting that when no tools are registered, the "Tool Meister" sub-system is not deployed and the bench scripts still execute normally.
+
+
+## All Tool Registration Handled Locally
+
+Along with the new "Tool Meister" sub-system comes another subtle, but significant, change to how tools are registered.
+
+With the v0.71 release, the record of which tools are registered on which hosts is kept local to the host on which `pbench-register-tool` or `pbench-register-tool-set` are invoked.
+
+Prior to v0.71, tool registration for remote hosts was recorded locally, and remotely via ssh.
+
+The registered tools are recorded in a local directory off of the "pbench_run" directory, by default `/var/lib/pbench-agent/tools-v1-<name>`, where `<name>` is the name of the Tool Group under which the tools were registered.
+
+The process of registering tools on local or remote hosts no longer validates that those tools are available during tool registration.  The Tool Meister sub-system now reports when registered tools are not present on registered hosts before beginning a benchmark run.  An error message will be displayed, and the particular "bench-script" will exit with a failure code.
+
+All tools registered prior to installing `v0.71.0-alpha` must be re-registered; tools registered locally or remotely on a host with v0.69 or earlier of the `pbench-agent` will be ignored.
+
+
+## New Support for Prometheus and PCP-based Tools
+
+The new "Tool Meister" sub-system enables support of Prometheus and PCP-based tools for data collection.
+
+The existing tools supported prior to the v0.71 release can be categorized as "Transient" tools.  By _transient_ we mean that a given tool is started and stopped immediately around the execution of a benchmark workload.  For example, when using `pbench-fio -b 4,16,32 -t read,write`, the transient tools are started immediately before each `fio` job is executed, and stopped immediately following its completion, for each of the 6 (six) `fio` jobs that would be run.
+
+A new category is introduced for Prometheus and PCP called "Persistent" tools. Persistent are started once at the beginning of a benchmark script, stopped at its end.  Using the previous `pbench-fio` example, persistent tools would be started before any of the 6 (six) `pbench-fio` jobs begin, and would be stopped once all six end.
```

Comment:
s/, stopped/and stopped/

Also, remove the second comma from the last sentence.